randovania/games/samus_returns/gui/dialog/game_export_dialog.py

```diff
@@ -95,7 +98,17 @@ def romfs_validation(line: QtWidgets.QLineEdit) -> bool:
     )
 
 
-#
+def exheader_validation(path: Path | None) -> bool:
+    if is_file_validator(path):
+        return True
+    path_typed = typing.cast(Path, path)  # this is always true because is_file_validator is true for None
```

Comment:
should be fine to just do `assert path is not None`, this isn't performance critical